event_response_tool/README.md

```diff
@@ -0,0 +1,106 @@
+# Event Response Tool
+
+## Overview
+
+The purpose of the Event Response Tool is to provide RMS style disaster response analysis in PRIME platform. It is based on the idea that the RMS users are provided with a list of events and associated weights that are meant for constructing a weighted convolution of event distribution into a single new event. For each event, the mean, standard deviation and exposure value attributes are then multiplied by their respective weight and then aggregated into a single event.
+
+To aid this type of analysis, the tool supports the following two functionalities:
+
+**1. Creating and Updating specialized Event Response Analysis Profile**
+**2. Transforming existing LayerViews and underlying ELTs**
+
+#### Analysis Profile
+The tool creates a specialized analysis profile that has the following components:
+
+- **Event Catalog**: A dummy event catalog with one or more EventIDs starting from 1.
+
+- **Simulation**: A simulation of size 'n' where n is the weighted sum of maximum trials per event.
+
+- **Loss Filters**: A set of loss filters representing every individual event in the event catalog.
+
+- **Exchange Rates Profile**: If the default FX profile UUID is not configured in the `config/event_response_config.ini` file,
+the latest FX profile available on the server.
+
+The tool also supports updating the simulation component of an existing event response analysis profile when the RMS users receive updated event weights. The user has to be provide the UUID of the existing analysis profile to the tool in order to use this feature.
+
+#### Transforming LayerViews and underlying ELTs
+
+When the user provide a CSV containing a list of LayerViews or a PortfolioView UUID with a set of LayerViews that are to be manipulated, the tool performs the following operations:
+
+- Iterate the LayerViews and retrieve the underlying ELTs.
+- Filter the ELTs based on the EventIDs in the event weights CSV file.  
+- Multiply the Mean, StdC, StdI and ExposureValue by the weight provided by RMS.
+- Change the EventID for each row to a single EventID in the event catalog.
```

Comment:
```suggestion
- Change the EventID for each row to an EventID in the event catalog.
```
Is this 1-to-1? `single` here implies many-to-1 for me.

Reply:
So, the intention here is actually many-to-1. All the events are aggregated to a single event. More info on methodology can be found here [https://analyzere.atlassian.net/wiki/spaces/AR/pages/2927067137/Toa+Event+Response+Documents+and+Interpretation](https://analyzere.atlassian.net/wiki/spaces/AR/pages/2927067137/Toa+Event+Response+Documents+and+Interpretation)